README.md

```diff
@@ -76,3 +76,7 @@ You can improve this page and help the Android community by opening [an issue](h
 # Google Play Services
 
 * Play Services is not open source. I added this entry because people keep asking about it 😅.
+
+# Android Open Source Project Code Review Tool
+
+* A web-based code review system for projects that use Git. Available [on android-review.googlesource.com](https://android-review.googlesource.com/q/status:open+-is:wip)
```

Comment:
Thanks! This link isn't the sources for the code review tool, but it's the tool itself. Not that its sources would all that be interesting, but it's more about the framing and how this differs from the other entries.

This site is interesting as it shows how one can contribute. But might be worth integrating direct editing / contributing links next to each project. Also, when I edit a file on cs.android.com it takes me to ci.android.com (and not android-review.googlesource.com, at least not immediately). I'm a bit confused which website does what.

Reply:
Oh yeah, that's the tool. I added because it could be helpful in the context of your repo.

Regarding the differences between ci.android.com and android-review.googlesource.com :

Quoting from https://source.android.com/setup/contribute/source-editor:

"Git Source Editor (ci.android.com/edit) is a browser-based tool for editing files in the Android Open Source Project (AOSP) and uploading changes to Gerrit. It’s intended for making minor changes to a small number of files, and provides a faster workflow than downloading the Android source tree. You should be familiar with the full process of submitting changes to AOSP using Git, Repo, and Gerrit before using Git Source Editor.

When you edit a file using Git Source Editor, all of the work happens on your local machine, until you upload changes to create a Gerrit changelist."

The doc https://source.android.com/setup/contribute/source-editor could help in understanding the difference.

Reply:
Interesting! Let me think about how we can incorporate all this.